src/main/java/at/technikum/tolanzeilinger/tourplanner/service/implementations/FolderOpenerService.java

```diff
@@ -0,0 +1,68 @@
+package at.technikum.tolanzeilinger.tourplanner.service.implementations;
+
+import at.technikum.tolanzeilinger.tourplanner.event.Event;
+import at.technikum.tolanzeilinger.tourplanner.event.EventAggregator;
+import at.technikum.tolanzeilinger.tourplanner.log.Logger;
+import at.technikum.tolanzeilinger.tourplanner.service.interfaces.PropertyLoaderService;
+import at.technikum.tolanzeilinger.tourplanner.service.interfaces.TourLogService;
+import at.technikum.tolanzeilinger.tourplanner.service.interfaces.TourService;
+import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import java.awt.Desktop;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FolderOpenerService {
```

Comment:
please create an interface here and implement it. important for service classes. And also update it in the ViewFactory

And rename this to "FolderOpenerServiceImpl" for naming conventions sake